File: openapi-test-kotlin-server-spring/testapi/model/Address.kt
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Optional

data class Address(
  @JsonProperty("city") val city: Optional<String>,
  @JsonProperty("country") val country: Optional<String>,
  @JsonProperty("street") val street: Optional<String>,
  @JsonProperty("zipCode") val zipCode: Optional<String>
)

File: openapi-test-scala-server-spring-client-jdk/testapi/model/Address.scala
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Optional

case class Address(
  @JsonProperty("city") city: Optional[String],
  @JsonProperty("country") country: Optional[String],
  @JsonProperty("street") street: Optional[String],
  @JsonProperty("zipCode") zipCode: Optional[String]
)